runtime/server/server.go

```diff
@@ -241,3 +246,49 @@ func (s *Server) Ping(ctx context.Context, req *runtimev1.PingRequest) (*runtime
 	}
 	return resp, nil
 }
+
+func InitOpenTelemetry() error {
+	otelAgentAddr, ok := os.LookupEnv("OTEL_EXPORTER_OTLP_ENDPOINT")
+	if !ok {
+		return nil
+	}
```

Comment:
Env vars should be parsed in `cli/cmd/runtime/start.go` into the `Config` and passed down from there when possible